### PR TITLE
Google Sign In Client Setup 

### PR DESCRIPTION
**Pull request by Jma353**
Hook up endpoint calls to client for Google Sign In, set a cookie for the accessToken we will use to authenticate requests, reflect basic information about the user via a `Profile.js` file.

### CODE REVIEW

**Comment by Jma353** [comment]
@AAAstorga @dantheli feel free to review this so we get used to PR flow.

File: client/src/components/Home.js [hunk 1]
```diff
@@ -1,19 +1,14 @@
 // @flow
+import ClickerPage from './common/ClickerPage';
 import React from 'react';
 
 class Home extends React.Component {
   render (): React.Element<any> {
     return (
-      <div style={styles.hello}>Hello, World!</div>
+      <ClickerPage>
+      </ClickerPage>
```
Comment (dantheli) [inline client/src/components/Home.js]: I assume this will have children later on? Other wise `<ClickerPage />`

Comment (Jma353) [reply client/src/components/Home.js]: Will have children later

**Comment by Jma353** [comment]
@dantheli

**Comment by Jma353** [comment]
Merging this to unblock myself

**Pull request merged**